
float64 time

float64 UAV_lat
float64 UAV_lon
float64 UAV_alt
float64 UAV_yaw
float64 UAV_vx
float64 UAV_vy
float64 cam_x
float64 cam_y
float64 cam_z

